README.md

```diff
@@ -1,6 +1,7 @@
 # Django secure redis
 Django caching plugin for django-redis that adds a Serializer class and configuration to support transparent,
-symmetrical encryption of cached values using the python cryptography library
+symmetrical encryption of cached values using the python cryptography library.
+This pluging also provide encryption for `django-rq` jobs by simply using `@secure_redis.rq_job` decorator on top of the job instead of `@django_rq.job`
```

Comment:
This plugin also provides encryption for `django-rq` jobs by simply using the `@secure_redis.rq_job` decorator to annotate the task method instead of using `@django_rq.job`